doside

```diff
@@ -36,12 +36,10 @@ tdump8000.py $mynode &
 LOGF=$HOME/VAR/logs/paris-traceroute.log
 rm -f $LOGF
 
-# Attempt to restart paris_rollins until /dev/shm/iupui_npad is mounted by the
+# Wait until /dev/shm/iupui_npad is mounted by the
 # vserver start sequence.
-while true; do
-  if grep iupui_npad /proc/mounts; then
-    paris_rollins.py -l $BASE/paris-traceroute >> $LOGF 2>&1 &
-    break
-  fi
+until grep iupui_npad /proc/mounts; do
```

Comment:
If we're changing this anyway, how about explicitly checking for complete expected `/dev/shm/iupui_npad` path here? In retrospect, it concerned me that we weren't making the check explicit here. i.e.

```
until grep /dev/shm/iupui_npad /proc/mounts; do
    sleep 1
done
```